ClemBot.Bot/bot/api/message_route.py

```diff
@@ -62,3 +64,11 @@ async def batch_edit_message(self, messages: t.Iterable, **kwargs):
 
     async def get_message(self, message_id: int):
         return await self._client.get(f'messages/{message_id}')
+
+    async def range_count_messages(self, user_id: int, guild_id: int, days: int):
```

Comment:
this function should return an int, the consumer of it doesnt want a dict with a number, just the number